src/cltkv1/wrappers/__init__.py

```diff
@@ -31,16 +31,14 @@ def __init__(self, data_input, language):
         """Constructor."""
         self.data_input = data_input
         self.language = language
-        self.nlp_doc_stanford = self._get_stanford_nlp_obj()
-        self.words = self.stanfordnlp_to_cltk_word_type()
+        self.stanfordnlp_wrapper = StanfordNLPWrapper(language=self.language)
 
-    def _get_stanford_nlp_obj(self):
-        """Call ``stanfordnlp`` and return original document object."""
-        nlp_obj_stanford = StanfordNLPWrapper(language=self.language)
-        return nlp_obj_stanford.parse(text=self.data_input)
+    def algorithm(self):
+        stanfordnlp_doc = self.stanfordnlp_wrapper.parse(self.data_input)
+        return = self.stanfordnlp_to_cltk_word_type(stanford_doc)
```

Comment:
Would like to see what you're thinking here. Since this won't run as is, should we wait and add your next additions to this PR?

Reply:
Sorry that was a temporary commit (switching computers).  Can you look now?  I haven't run the full tests but at least one example runs.

Reply:
@kylepjohnson all tests pass now.